statistics/handle/handle.go

```diff
@@ -1112,17 +1112,18 @@ func (h *Handle) histogramFromStorage(reader *statsReader, tableID int64, colID
 }
 
 func (h *Handle) columnCountFromStorage(reader *statsReader, tableID, colID, statsVer int64) (int64, error) {
+	count := int64(0)
 	rows, _, err := reader.read("select sum(count) from mysql.stats_buckets where table_id = %? and is_index = 0 and hist_id = %?", tableID, colID)
 	if err != nil {
 		return 0, errors.Trace(err)
 	}
-	if rows[0].IsNull(0) {
-		return 0, nil
-	}
-	count, err := rows[0].GetMyDecimal(0).ToInt()
-	if err != nil {
-		return 0, errors.Trace(err)
+	if !rows[0].IsNull(0) {
```

Comment:
We may need to add some comments. From the codes, we can see that we've modified the codes to be adjusted with Version2 but still found something wrong this time.

Reply:
Added some comments. PTAL again.